### ISSUE TITLE
Met bugs ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: -9) local_rank: 0

### ISSUE THREAD

**Issue by saleelirenyun** [pytorch/pytorch#82756]
### 🐛 Describe the bug

I met bugs when I run distribute train. 
![image](https://user-images.githubusercontent.com/42291628/182712008-0389906c-84be-470d-9f49-bfa86d861cd0.png)

Package            Version
------------------ ----------------
certifi            2022.6.15
charset-normalizer 2.1.0
idna               3.3
numpy              1.23.1
Pillow             9.2.0
pip                22.1.2
PyYAML             6.0
requests           2.28.1
scipy              1.9.0
setuptools         61.2.0
timm               0.4.5
tlt                0.1.0
torch              1.12.0+rocm5.1.1
torchaudio         0.12.0+rocm5.1.1
torchvision        0.13.0+rocm5.1.1
typing_extensions  4.3.0
urllib3            1.26.11
wheel              0.37.1

### Versions

Collecting environment information...
PyTorch version: 1.12.0+rocm5.1.1
Is debug build: False
CUDA used to build PyTorch: N/A
ROCM used to build PyTorch: 5.1.20531-cacfa990

OS: Red Hat Enterprise Linux release 8.4 (Ootpa) (x86_64)
GCC version: (GCC) 8.4.1 20200928 (Red Hat 8.4.1-1)
Clang version: Could not collect
CMake version: version 3.18.2
Libc version: glibc-2.28

Python version: 3.9.12 (main, Jun  1 2022, 11:38:51)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-4.18.0-305.28.1.el8_4.x86_64-x86_64-with-glibc2.28
Is CUDA available: True
CUDA runtime version: Could not collect
GPU models and configuration: 
Nvidia driver version: Could not collect
cuDNN version: Could not collect
HIP runtime version: 5.1.20531
MIOpen runtime version: 2.16.0
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.23.1
[pip3] torch==1.12.0+rocm5.1.1
[pip3] torchaudio==0.12.0+rocm5.1.1
[pip3] torchvision==0.13.0+rocm5.1.1
[conda] numpy                     1.23.1                   pypi_0    pypi
[conda] torch                     1.12.0+rocm5.1.1          pypi_0    pypi
[conda] torchaudio                0.12.0+rocm5.1.1          pypi_0    pypi
[conda] torchvision               0.13.0+rocm5.1.1          pypi_0    pypi

**Comment by d4l3k** [comment]
closing as stale issue, please reopen or create a new issue if you're still experiencing this
